recipes/opportunity_insights/runner.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+source $(pwd)/bin/config.sh
+BASEDIR=$(dirname $0)
+VERSION=$DATE
+TYPE=$1 #weekly/daily
+
+mkdir -p input
+mkdir -p output
```

Comment:
this would create the input/output folder in the root directory, we would need to move them into the blocks under `cd $BASEDIR`